src/Components/CreateImageWizard/validators.ts

```diff
@@ -51,6 +53,20 @@ export const isBlueprintNameValid = (blueprintName: string) =>
   blueprintName.length <= 100 &&
   /\w+/.test(blueprintName);
 
+export const generateDefaultName = (
+  distribution: string,
+  arch: string,
+  targetEnvironments: ImageTypes[]
+) => {
+  const date = new Date();
+  const day = date.getDate().toString().padStart(2, '0');
+  const month = date.toLocaleString('en-US', { month: 'long' });
+  const year = date.getFullYear();
+  return `${distribution} ${arch} ${targetEnvironments.join(
+    ' '
+  )} ${day} ${month} ${year}`;
+};
+
```

Comment:
I don't think this function should be in validators. We can maybe create a separate `generateDefaultName.ts` file for it in `src/CreateImageWizard/utilities`?

Reply:
thank you I think it is a good solution!